app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss

```diff
@@ -295,102 +304,90 @@ $after-button-padding-left: govuk-spacing(5);
     padding: govuk-spacing(3) 0;
   }
 
+  @include govuk-media-query($until: "desktop") {
+    position: absolute;
+    right: 34px;
```

Comment:
I do wonder whether instead of using absolutely positioning elements and having to set their exact locations in pixels we could align the logo, the menu button and the search button using `display: inline-block`. Currently they're positioned with a combination of floats and absolutely positioning and magicky numbers that could potentially be a bit brittle. However the existing navbar already does this so I guess not something we need to fix this in this PR.